test/e2e/specs/specs-playwright/wp-invite__new-user.ts

```diff
@@ -12,7 +16,7 @@ import {
 } from '@automattic/calypso-e2e';
 import { Page } from 'playwright';
 
-describe.skip( DataHelper.createSuiteTitle( `Invite: New User` ), function () {
+describe( DataHelper.createSuiteTitle( `Invite: New User` ), function () {
```

Comment:
Is it intended to un-skip this one within this PR?

Reply:
Yep, I added the proper Jest test group headers to the file.

Reply:
I've triggered an instance of the pre-release tests with this test enabled - if it the test has issues, I will remove this change from the PR and address it separately.

Reply:
I'll remove this change from this PR, since it isn't explicitly tied to the refactoring of LoginFlow.